alerter/build.gradle

```diff
@@ -6,14 +6,12 @@ plugins {
 apply plugin: 'com.android.library'
 apply plugin: 'maven'
 apply plugin: "maven-publish"
+apply plugin: 'com.tapadoo.android.sputnik'
```

Comment:
Thats the point of Sputnik though, to have a single source for all checkstyles, lint etc. rules. If there are some rules/suppressions that need updating, they should be pushed to Sputnik too.